app/models/vm_scan.rb

```diff
@@ -600,6 +556,32 @@ def abort_retry(*args)
 
   private
 
+  def create_snapshot(vm)
```

Comment:
@bdunne the caller is `return unless create_snapshot(vm)`.  If we have returns without values inside the method, how would the caller's unless get triggered?